golos.publication/golos.publication.cpp

```diff
@@ -826,10 +832,14 @@ int64_t publication::pay_delegators(int64_t claim, name voter,
     int64_t dlg_payout_sum = 0;
     for (auto delegate_obj : delegate_list) {
         auto dlg_payout = claim * delegate_obj.interest_rate / config::_100percent;
-        INLINE_ACTION_SENDER(golos::vesting, paydelegator) (config::vesting_name,
-            {config::vesting_name, config::active_name},
-            {voter, eosio::asset(dlg_payout, tokensymbol), delegate_obj.delegator,
-            delegate_obj.payout_strategy});
+        if (golos::vesting::balance_exist(config::vesting_name, delegate_obj.delegator, tokensymbol.code()))
```

Comment:
What funds is this payment made (token or vesting)? Which tests checks that this code work correctly? I have concerns that this code is not working properly. And tests for it were not implemented (see #449)